mediator/src/middlewares/schemas/patient.ts

```diff
@@ -8,6 +8,6 @@ export const createPatientSchema = joi.object({
   id: joi.string().optional(),
   sex: joi.string().trim().valid('male', 'female', 'other', 'unkown').required(),
   // eslint-disable-next-line camelcase
-  date_of_birth: joi.string().optional(),
+  date_of_birth: joi.date().iso().required(),
```

Comment:
@njogz I don't know if date is required, but it looks like it is required. And is there anyway of validating date in Joi without being strictly iso standard? Thanks in advance.